src/minType.js

```diff
@@ -299,6 +299,22 @@ function minType (sup, sub) {
     return consistencyCheck(computed)
   }
 
+  if (subType === 'union' && superType !== 'union') {
+    let anyOf = sub.anyOf
+    let unionResult = sub
+    unionResult.anyOf = []
+    for (let i = 0; i < anyOf.length; i++) {
+      let result = minType(anyOf[i], sup)
```

Comment:
Should these arguments be reversed? The signature is `minType(sup, sub)`.

Reply:
Fixed